### PR TITLE
Integrate Sentry

### PR DESCRIPTION
**Pull request by doitian**
#### Show file location in release profile

Enable debug for release profile in `Cargo.toml`
    
```
[profile.release]
debug = true
```

or via `RUSTFLAGS`

```
RUSTFLAGS=-g cargo build --release
```

#### Register panic hook

```
sentry::integrations::panic::register_panic_handler()
```

####  Set release info

```
let sentry = sentry::init(sentry::ClientOptions {
    release: Some("foo-bar-baz@1.0.0".into()),
    dsn: "https://48c6a88d92e246478e2d53b5917a887c@sentry.io/1422795"
        .parse()
        .ok(),
    ..Default::default()
});
if sentry.is_enabled() {
    sentry::integrations::panic::register_panic_handler();
}
```

- - - -
##### References

- [Getting Started with Rust Error Tracking](https://blog.sentry.io/2018/10/22/getting-started-with-rust-error-tracking)

### CODE REVIEW

File: test/fixtures/nodes_template/default.toml [hunk 1]
```diff
@@ -45,3 +45,7 @@ txs_verify_cache_size = 100000
 [block_assembler]
 # type hash for `always_success` script
 type_hash = "0x8954a4ac5e5c33eb7aa8bb91e0a000179708157729859bd8cf7e2278e1e12980"
+
+[sentry]
+# set to blank to disable sentry error collection
+dsn = "https://48c6a88d92e246478e2d53b5917a887c@sentry.io/1422795"
```
Comment (quake) [inline test/fixtures/nodes_template/default.toml]: I think it's better to disable sentry in integration test.

**Comment by janx** [comment]
Can you also update the README to notify users we're using Sentry to collecting stats in testnet stage?

File: README.md [hunk 2]
```diff
@@ -13,6 +13,10 @@ Nervos CKB is the layer 1 of Nervos Network, a public blockchain with PoW and ce
 
 Nervos project defines a suite of scalable and interoperable blockchain protocols. Nervos CKB uses those protocols to create a self-evolving distributed network with a novel economic model, data model and more.
 
+**Warning**: The ckb process will send stack trace to sentry on Rust panics.
```
Comment (janx) [inline README.md]:
```suggestion
**Notice**: The ckb process will send stack trace to sentry on Rust panics.
```

**Comment by doitian** [comment]
bors r=quake

**Comment by doitian** [comment]
bors retry

**Pull request merged**